config/checks/run-checks.bat

```diff
@@ -0,0 +1,12 @@
+@echo off
+rem Runs all check-*.bat scripts, and returns a non-zero exit code if any of them fail.
+
+setlocal enabledelayedexpansion
+set ret=0
+
+for %%X in (check-*.bat) do (
+ call "%%X"
+ if ERRORLEVEL 1 set ret=1
+)
+echo bye
```

Comment:
Is this intentional?

Reply:
That was not meant to be committed, but was only intended for testing purposes, sorry about that

Reply:
No problem, was just a little shocked when the script said 'bye' after I ran it 😂